db/schema.rb

```diff
@@ -258,7 +258,9 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "status", default: "Missing"
-    t.integer "exposure_to_isolation"
+    t.integer "exposure_to_isolation_active"
+    t.integer "exposure_to_isolation_not_active"
+    t.integer "cases_closed_in_exposure"
```

Comment:
Should these be here, or did they sneak in from some funny business with #1085?

Reply:
ah good catch, I think these snuck in here because I didn't migrate down after reviewing that PR

Reply:
this has been fixed